<launch>

  <!-- Experiment Playback -->
<param name ="/use_sim_time" value="true"/>  
<node pkg="rosbag" type="play" name="player" output="screen" 
args="--clock -r 0.05 -s 7 -u 7 /mnt/hgfs/Experiment-Data-MIT/test/bucket_video_test_2017-12-06-18-49-56.bag"/>

<!-- Mouse Click Test Node -->
<!--
<node pkg="opt_flow" type="mouse_click_test.py" name="mouse_click_test" output="screen">
        <remap from="UBC_Image"  to="/camera1/image_raw" />
	<rosparam>
     </rosparam>
</node> -->

<!-- Crop Image Node -->

<node pkg="opt_flow" type="img_crop.py" name="image_crop" output="screen">
        <remap from="UBC_Image"  to="/camera1/image_raw" />
	<remap from="cal_Data"  to="/CalibratedJoints" />
	<param name="Filename" value="data_1212.txt" />
        <rosparam>
     </rosparam>
</node>  


  <!-- Record Optical Flow Stats -->
<!-- <node pkg="rosbag" type="record" name="flow_recorder" args="record -o /home/weitung/Documents/Experiment_Data/1017/ /Flow_Image"/> -->	
</launch>
